<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajcaClass</class>
 <widget class="QMainWindow" name="MajcaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Majca</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Euler</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Midpoint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Taylor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rungego-Kutty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ile rekord√≥w</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Co ile</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="leCount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="leStart">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="leCoIle">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pltEuler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pltMidpoint">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pltTaylor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pltRungegoKutty">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Majca.qrc"/>
 </resources>
 <connections/>
</ui>
